lib/peep.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+
+class Peep
+  attr_reader :message, :date
+
+  def initialize(message:, date:)
+    @message = message
+    @date = date
+  end
+
+  def self.all
```

Comment:
You could perhaps extract the code that checks if you are in a test environment to another file in a function and just reference it here, to avoid repeating.